lms/djangoapps/courseware/module_render.py

```diff
@@ -138,7 +138,9 @@ def toc_for_course(request, course, active_chapter, active_section, field_data_c
     '''
```

Comment:
testing module_render,py
14 FAILED, 34 errors (67 tests passed)
I think test need to be update due t change in module.render.py